Terminal.Gui/Drawing/Color.cs

```diff
@@ -1,369 +1,155 @@
+	public Color ( int rgba )
 	{
-		R = red;
-		G = green;
-		B = blue;
-		A = alpha;
+		Rgba = rgba;
 	}
 
 	/// <summary>
-	/// Initializes a new instance of the <see cref="Color"/> class with an encoded 24-bit color value.
+	/// Initializes a new instance of the <see cref="Color"/> class with an encoded unsigned 32-bit color value in ARGB32 format.
 	/// </summary>
-	/// <param name="rgba">The encoded 24-bit color value (see <see cref="Rgba"/>).</param>
-	public Color (int rgba)
+	/// <param name="argb">The encoded unsigned 32-bit color value (see <see cref="Argb"/>).</param>
+	/// <remarks>The alpha channel is not currently supported, so the value of the alpha channel bits will not affect rendering.</remarks>
+	public Color ( uint argb )
 	{
-		A = (byte)(rgba >> 24 & 0xFF);
-		R = (byte)(rgba >> 16 & 0xFF);
-		G = (byte)(rgba >> 8 & 0xFF);
-		B = (byte)(rgba & 0xFF);
+		Argb = argb;
 	}
 
 	/// <summary>
-	/// Initializes a new instance of the <see cref="Color"/> color from a legacy 16-color value.
+	/// Initializes a new instance of the <see cref="Color"/> color from a legacy 16-color named value.
 	/// </summary>
 	/// <param name="colorName">The 16-color value.</param>
-	public Color (ColorName colorName)
+	public Color ( in ColorName colorName )
 	{
-		var c = FromColorName (colorName);
-		R = c.R;
-		G = c.G;
-		B = c.B;
-		A = c.A;
+		this = ColorExtensions.ColorNameToColorMap [ colorName ];
 	}
 
 	/// <summary>
-	/// Initializes a new instance of the <see cref="Color"/> color from string. See <see cref="TryParse(string, out Color)"/>
+	/// Initializes a new instance of the <see cref="Color"/> color from string. See <see cref="TryParse(string, out Color?)"/>
 	/// for details.
 	/// </summary>
 	/// <param name="colorString"></param>
-	/// <exception cref="Exception"></exception>
-	public Color (string colorString)
+	/// <exception cref="ArgumentNullException">If <paramref name="colorString"/> is <see langword="null"/>.</exception>
+	/// <exception cref="ArgumentException">If <paramref name="colorString"/> is an empty string or consists of only whitespace characters.</exception>
+	/// <exception cref="ColorParseException">If thrown by <see cref="Parse(string?,System.IFormatProvider?)"/></exception>
+	public Color ( string colorString )
 	{
-		if (!TryParse (colorString, out var c)) {
-			throw new ArgumentOutOfRangeException (nameof (colorString));
-		}
-		R = c.R;
-		G = c.G;
-		B = c.B;
-		A = c.A;
+		ArgumentException.ThrowIfNullOrWhiteSpace ( colorString, nameof ( colorString ) );
+		this = Parse ( colorString, CultureInfo.InvariantCulture );
 	}
 
 	/// <summary>
-	/// Initializes a new instance of the <see cref="Color"/>.
+	/// Initializes a new instance of the <see cref="Color"/> with all channels set to 0.
 	/// </summary>
-	public Color ()
+	public Color ( )
 	{
-		R = 0;
-		G = 0;
-		B = 0;
-		A = 0xFF;
+		Argb = 0u;
 	}
 
+    /// <summary>
```

Comment:
Hit CTRL-K CTRL-D and this will get fixed.

Reply:
Not sure what, specifically, you meant to point out here.

Hopefully it's handled in the upcoming change set, but let me know if otherwise.